buildkite/pipeline.yml

```diff
@@ -18,6 +20,8 @@ steps:
 
   - label: ":codecov: Codecov"
     command: docker/coverage
+    agents:
+      queue: "parallel"
```

Comment:
as the number of things that wants to use this queue increases, overall time will go up. Any thoughts on how we should budget runners going forward?

Reply:
Thinking we probably need some monitoring on the buildkite agents. Some idea of load, memory usage and idle time would be quite handy as a start so we can see if we can reduce the number of standard agents or if we need more parallel agents. I expect that in reality we have fairly spiky load where we might be using all agents for a short amount of time (e.g. because we have some builds which run many things in parallel) which might be quite challenging to handle. We don't want to end up with a bunch of buildkite agents sitting on all the resources on the server and being idle 95% of the time. I think getting some metrics would be a good start